angular-ngrx-scss/src/app/state/auth/auth.effects.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { of } from 'rxjs';
+import { catchError, concatMap, map, take, tap } from 'rxjs/operators';
+import { loadUserToken, loadUserTokenFailure, loadUserTokenSuccess } from '.';
+import { AuthService } from '../../auth/services/auth.service';
+import { startSignIn } from './auth.actions';
+
+@Injectable()
+export class AuthEffects {
+  signIn$ = createEffect(() => {
+    return this.actions$.pipe(
+      ofType(startSignIn),
+      tap(() => this.authService.signIn()),
+      take(1),
```

Comment:
Removing the `take(1)` was causing an infinite loop - instead, since this effect only needs to trigger the service function call and then be done, set it to not dispatch which both prevents the infinite looping and allows for multiple calls to this function when needed.